### PR TITLE
Newest code conventions implemented

### PR DESCRIPTION
**Pull request by omrfdk**

### CODE REVIEW

File: src/SBarcodeFilter.h [hunk 1]
```diff
@@ -8,48 +8,123 @@
...
+    /*!
+     * \fn void clean()
+     * \brief Sets the captured barcode string to empty.
+     */
     void clean();
 
 private:
     QString _captured = "";
+
     QRectF _captureRect;
```
Comment (j-kies) [inline src/SBarcodeFilter.h:121]: C++ code rule 1 for member variables: `camelCase starting with 'm_', like std::string m_accessToken;.`

File: src/SBarcodeGenerator.h [hunk 2]
```diff
@@ -25,43 +28,116 @@ class SBarcodeGenerator : public QQuickItem
...
+     */
     void formatChanged(SCodes::SBarcodeFormat format);
 
 private:
-    int _width = 500;
-    int _height = 500;
-    int _margin = 10;
+    int _width    = 500;
+    int _height   = 500;
+    int _margin   = 10;
```
Comment (j-kies) [inline src/SBarcodeGenerator.h:132]: C++ code rule 1 for member variables: `camelCase starting with 'm_', like std::string m_accessToken;.`

File: src/SBarcodeDecoder.cpp [hunk 3]
```diff
@@ -39,47 +38,52 @@ class Result : private ZXing::Result
...
-        default: return ImageFormat::None;
-        }
-    };
+    auto ImgFmtFromQImg = [](const QImage& img){
+          switch (img.format()) {
+              case QImage::Format_ARGB32:
+              case QImage::Format_RGB32:
+                  #if Q_BYTE_ORDER == Q_LITTLE_ENDIAN
+                  return ImageFormat::BGRX;
+
```
[the remark this answers is not in the document]
Comment (omrfdk) [reply src/SBarcodeDecoder.cpp:49]: uncrustify making this

File: src/SBarcodeDecoder.cpp [hunk 4]
```diff
@@ -39,47 +38,52 @@ class Result : private ZXing::Result
...
+                  #else
+                  return ImageFormat::XRGB;
+
```
[the remark this answers is not in the document]
Comment (omrfdk) [reply src/SBarcodeDecoder.cpp:52]: uncrustify making this

File: src/SBarcodeDecoder.cpp [hunk 5]
```diff
@@ -39,47 +38,52 @@ class Result : private ZXing::Result
...
+                  #endif
+              case QImage::Format_RGB888: return ImageFormat::RGB;
 
-    auto exec = [&](const QImage& img) {
-        return Result(ZXing::ReadBarcode({img.bits(), img.width(), img.height(), ImgFmtFromQImg(img)}, hints));
-    };
+              case QImage::Format_RGBX8888:
+              case QImage::Format_RGBA8888: return ImageFormat::RGBX;
+
```
[the remark this answers is not in the document]
Comment (omrfdk) [reply src/SBarcodeDecoder.cpp:58]: uncrustify making this

**Comment by omrfdk** [comment]
done in the new PR

**Pull request closed without merging**